### PR TITLE
Add content from the Stolen Fate Player's Guide

### PR DESCRIPTION
**Pull request by TikaelSol**

### CODE REVIEW

File: packs/data/feat-effects.db/effect-harrow-the-fiend.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "target:trait:fiend"
+                ],
+                "selector": "attack",
```
Comment (simonward) [inline packs/data/feat-effects.db/effect-harrow-the-fiend.json:25]: This should probably be:
```
"selector": [
	"spell-attack-roll",
	"strike-attack-roll"
]
```

**Comment by simonward** [comment]
The source should probably be "Pathfinder: Stolen Fate Player's Guide" to match the others.

File: packs/data/backgrounds.db/shielded-fortune.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    "dex",
+                    "int",
+                    "str",
+                    "wis"
+                ]
+            }
+        },
+        "description": {
+            "value": "<p>You've always been tougher than your peers. As a child, you may have been able to endure the rough-and-tumble horseplay with friends more easily. You regularly emerged relatively unscathed from accidents could have been quite dangerous. At some point in your past, you had a particularly close call with death—be it from a near-fatal accident, a fight gone horribly wrong, a close call with illness, or something similar, yet you quickly recovered. You may or may not believe you're more fated to face unexpected hardship in your life, but regardless, you know you can take what destiny has to dish out.</p>\n<p>This background is associated with the suit of Shields.</p>\n<p>Choose two ability boosts. One must be to <strong>Constitution</strong>, and one is a free ability boost. You gain the @UUID[Compendium.pf2e.feats-srd.Toughness]{Toughness} general feat and also gain the @UUID[Compendium.pf2e.actionspf2e.Fated Note to Die]{Fated Note to Die} free action.</p>"
```
Comment (simonward) [inline packs/data/backgrounds.db/shielded-fortune.json:24]: Missing word?
"unscathed from accidents ^ could have been"

File: packs/data/feats.db/harrower-dedication.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "class",
+        "description": {
+            "value": "<p>You become trained in Occultism; if you are already trained in this skill, you become trained in a different skill of your choice instead. You also become trained in Fortune-Telling Lore, and you learn the <em>@UUID[Compendium.pf2e.spells-srd.Harrowing]{Harrowing}</em> ritual.</p>\n<p>In addition, once a day when you perform your daily preparations, you can perform a harrow reading for yourself if you have access to a harrow deck. Draw a card from your harrow deck—that card's suit becomes your active harrow omen for the next 24 hours and modifies several other harrower feats as detailed on the following pages. You gain the @UUID[Compendium.pf2e.actionspf2e.Avoid Dire Fate]{Avoid Dire Fate} reaction.</p>\n<p>@UUID[Compendium.pf2e.feat-effects.Effect: Harrow Omen]{Effect: Harrow Omen}</p>\n<p><strong>Special</strong> You can't select another dedication feat until you've gained two other feats from the harrower archetype. If you retrain out of this feat, you lose the harrowing ritual you learned from it</p>"
```
Comment (simonward) [inline packs/data/feats.db/harrower-dedication.json:14]: Missing ending period.

File: packs/data/feat-effects.db/effect-harrow-the-fiend.json (hunk 1 above)
Comment (TikaelSol) [reply packs/data/feat-effects.db/effect-harrow-the-fiend.json:25]: No, just `attack` covers both, see inspire courage

**Pull request merged**